src/main/java/com/juick/android/ui/widget/CropSheet.kt

```
@@ -0,0 +1,137 @@
+                }
+
+                TextButton(
+                    onClick = {
+                        if (isCropping) return@TextButton
+                        isCropping = true
+                        cropImageView?.setOnCropImageCompleteListener { _, result ->
+                            isCropping = false
+                            val uri = if (result.isSuccessful) {
+                                try {
+                                    saveBitmapToFile(context, result.bitmap)
+                                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)